src/member-api.js

```diff
@@ -51,19 +56,38 @@ export class MemberApi extends TypedEmitter {
    *
    * @param {Object} opts
    * @param {import('./capabilities.js').RoleId} opts.roleId
+   * @param {string} [opts.roleName]
+   * @param {string} [opts.roleDescription]
    * @param {number} [opts.timeout]
    *
    * @returns {Promise<import('./generated/rpc.js').InviteResponse_Decision>}
    */
-  async invite(deviceId, { roleId, timeout }) {
-    const projectId = projectKeyToId(this.#projectKey)
+  async invite(deviceId, { roleId, roleName, roleDescription, timeout }) {
+    if (!(roleId === MEMBER_ROLE_ID || roleId === COORDINATOR_ROLE_ID)) {
+      throw new Error('Invalid role id')
+    }
```

Comment:
This will break some tests that invite a user as a blocked user. Could we maybe do:

```suggestion
    if (!(DEFAULT_CAPABILITIES[roleId])) {
      throw new Error('Invalid role id')
    }
```

Reply:
I think that's probably correct, however we didn't restrict this previously and in the e2e tests for sync authentication I invite as a blocked user to check that the blocked user cannot sync. Once we have the api for changing someone's role then we can update the tests, but maybe make this restriction looser for now to avoid lots of work on the sync tests and role api?